test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,36 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        
+        // all objects non null
+        assertNoNull();
+        assertNoNull((Object) 5);
+        assertNoNull(1);
+        assertNoNull("");
+        assertNoNull("abc");
+        assertNoNull("abc", "ab", "a");
+        assertNoNull(1, 2);
+
+        // some null objects
+        assertHasNull((Object) null);
+        assertHasNull("abc", "a", null);
+        assertHasNull(null, "", null, "ABC");
+        assertHasNull(null, Integer.valueOf(1));
+        assertHasNull(null, 1, Integer.valueOf(1));
+        assertHasNull(null, null);
+        assertHasNull("a", "b", null);
+        
+    }
+
+    private void assertNoNull(Object... objects) {
```

Comment:
Extracting the assert function is not really necessary here since this method is quite simple (with only one line). Thus, you can add it directly into your test cases.